DataFormats/CTPPSDigi/interface/CTPPSPixelDigi.h

```diff
@@ -0,0 +1,69 @@
+#ifndef CTPPS_PIXEL_DIGI_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h